prody/atomic/functions.py

```diff
@@ -300,4 +303,58 @@ def sliceAtoms(atoms, select):
         which = array([i for i, idx in enumerate(atoms._getIndices())
                           if idx in idxset])
 
-    return which, select
\ No newline at end of file
+    return which, select
+
+def sliceAtomicData(data, atoms=None, selection=None, axis=0):
+    """Slice a matrix using indices extracted using sliceAtoms.
+
+    :arg matrix: any matrix (2D array)
+    :type matrix: `~numpy.ndarray`
+
+    :arg selection: a :class:`Selection` instance or selection string. 
+    :type selection: :class:`Selection`, str
+
+    :arg axis: the axis/direction you want to use to slice data from the matrix.
+        The options are 0 or 1 like in `~numpy` or 'both'. Default is 0 (row).
```

Comment:
`~numpy` -> :mod:`~numpy`, also numpy uses `None` to mean "both", we may want to also make it consistent.